### PR TITLE
Add Immersive Movement

### PR DESCRIPTION
**Pull request by sean-kang**
I added metadata for Immersive Movement mod.
https://www.nexusmods.com/skyrimspecialedition/mods/15649?tab=description

Thanks.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -6769,6 +6769,47 @@ plugins:
       # version: 1.06a
       - crc: 0x20AA5984
         util: 'SSEEdit v3.2.2'
+  - name: 'Immersive Movement.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: Overall placement generally doesn't make a difference, but I'd prefer if this were placed somewhere more appropriate. If you're unsure, then just place new entries at the end of the masterlist.

File: masterlist.yaml [hunk 2]
```diff
@@ -6769,6 +6769,47 @@ plugins:
...
+    group: *lowPriorityGroup
```
Comment (MacSplody) [inline masterlist.yaml]: This group assignment isn't necessary and will cause sorting errors.

File: masterlist.yaml [hunk 3]
```diff
@@ -6769,6 +6769,47 @@ plugins:
...
+        subs: [ 'Wildcat' ]
+        condition: 'active("Wildcat - Combat of Skyrim.esp") and not active("Immersive Movement - Wildcat Patch.esp")'
+    clean:
+      # version: 1.3.2
+      - crc: 0xB91C4616
+        util: 'SSEEdit v4.0.2'
+  - name: 'Immersive Movement - Mortal Enemies Patch.esp'
+    after:
+      - 'Immersive Movement.esp'
+      - 'Mortal Enemies.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: An after is fine for Mortal Enemies but all Mortal enemy variants should be added.

`Mortal Enemies.esp`
`Mortal Enemies - No RunWalk Changes.esp`
`Mortal Enemies - Rival Remix.esp`
`Mortal Enemies - Rival Remix - NoRunWalkChanges.esp`
`Mortal Enemies - 2.0 Test Version.esp`

File: masterlist.yaml [hunk 4]
```diff
@@ -6769,6 +6769,47 @@ plugins:
...
+      - <<: *patchIncluded
+        subs: [ 'Wildcat' ]
+        condition: 'active("Wildcat - Combat of Skyrim.esp") and not active("Immersive Movement - Wildcat Patch.esp")'
+    clean:
+      # version: 1.3.2
+      - crc: 0xB91C4616
+        util: 'SSEEdit v4.0.2'
+  - name: 'Immersive Movement - Mortal Enemies Patch.esp'
+    after:
+      - 'Immersive Movement.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: `Req:` would be better for `Immersive Movement.esp`.

File: masterlist.yaml [hunk 5]
```diff
@@ -6769,6 +6769,47 @@ plugins:
...
+      - 'Mortal Enemies.esp'
+    clean:
+      # version: 1.3.2
+      - crc: 0xDCF50B37
+        util: 'SSEEdit v4.0.2'
+  - name: 'Immersive Movement - Smilodon Patch.esp'
+    after:
+      - 'Immersive Movement.esp'
+      - 'Smilodon - Combat of Skyrim.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: You can use `Req:` for both of these as well.

File: masterlist.yaml [hunk 6]
```diff
@@ -6769,6 +6769,47 @@ plugins:
...
+    clean:
+      # version: 1.3.2
+      - crc: 0x9906E4A8
+        util: 'SSEEdit v4.0.2'
+  - name: 'Immersive Movement - Wildcat Patch.esp'
+    after:
+      - 'Immersive Movement.esp'
+      - 'Wildcat - Combat of Skyrim.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: Same `Req:` for both of these as well.

**Review by MacSplody — CHANGES_REQUESTED** [review]

File: masterlist.yaml (hunk 2 above)
Comment (sean-kang) [reply masterlist.yaml]: Removed.

File: masterlist.yaml (hunk 5 above)
Comment (sean-kang) [reply masterlist.yaml]: Done.

File: masterlist.yaml (hunk 6 above)
Comment (sean-kang) [reply masterlist.yaml]: Done.

File: masterlist.yaml (hunk 4 above)
Comment (sean-kang) [reply masterlist.yaml]: Done.

File: masterlist.yaml (hunk 3 above)
Comment (sean-kang) [reply masterlist.yaml]: Done. Also added the variants to the condition.

File: masterlist.yaml (hunk 1 above)
Comment (sean-kang) [reply masterlist.yaml]: Moved to Gameplay section.

**Comment by sean-kang** [comment]
Thanks for the comments. I just pushed a new commit to fix them.

**Review by MacSplody — APPROVED** [review]
Nice work, Thanks.

**Comment by MacSplody** [comment]
@sean-kang Thanks, congrats on your first contribution.

**Comment by MacSplody** [comment]
#497

**Pull request merged**